src/Microsoft.ML.Data/Commands/DataCommand.cs

```diff
@@ -170,9 +170,9 @@ protected void SendTelemetryMetric(Dictionary<string, IDataView>[] metricValues)
                             {
                                 var nameOfMetric = "TLC_" + cursor.Schema[currentIndex].Name;
```

Comment:
>TLC_ [](start = 52, length = 4)

Do we need to change this? not that I protest! :)

Reply:
Change the "TLC" part? There are still a bunch of things with "TLC" in them... Maybe open an issue to do a mass "TLC" removal/rename?

Note this is part of the "telemetry" code, which is only invoked from `Command` classes, which are only invoked from MAML, as far as I can tell.